.changeset/four-months-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-backend': patch
+---
+
+Removed deprecated code
```

Comment:
Still a breaking change, even if it's deprecated

Reply:
Yes, updated, also improved the description, this was a bit too short IMO.